test.js

```diff
@@ -220,6 +238,46 @@ test('sync (ancestor directory, custom cwd)', t => {
 	t.is(foundPath, absolute.fixtureDirectory);
 });
 
+test('async (absolute directory)', async t => {
+	const filePath = await findUp(absolute.barDir);
+
+	t.is(filePath, absolute.barDir);
+});
+
+test('sync (absolute directory)', t => {
+	const filePath = findUp.sync(absolute.barDir);
+
+	t.is(filePath, absolute.barDir);
+});
+
+test('async (not found, absolute directory)', async t => {
```

Comment:
I think you meant to write "absolute file"? (Same with the sync test.)

Reply:
fixed.